site/content/recipes/review_meetings.md

```diff
@@ -1,8 +1,85 @@
 # Perform code review
 Draft outline base on  [flowcharts/high-level.md](https://github.com/ResearchCodeReviewCommunity/dev-review/blob/main/site/content/flowcharts/high-level.md#perform-code-review)
+
+This is [recipes/review_meetings.md](https://github.com/ResearchCodeReviewCommunity/dev-review/blob/main/site/content/recipes/review_meetings.md). You can edit the file normally (locally) and push to the GitHub remote to collaborate.
```

Comment:
same here: this is text from HackMD that we can remove? Or is there something I'm not understanding?